kedro-datasets/kedro_datasets/huggingface/transformer.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any
+from kedro.io import AbstractDataset
+from transformers import AutoTokenizer, AutoModel
+
+import logging
+import importlib
+
+from collections import namedtuple
+
+
+TransformerModel = namedtuple("TransformerModel", ["model", "tokenizer"])
+
+
+logger = logging.getLogger(__file__)
+
+
+class HFTransformer(AbstractDataset):
```

Comment:
What about `HFAutoTransformerDataset`? 

(About `...Dataset`, I know it's not technically a _dataset_ and that there's potential confusion with [Hugging Face Datasets](https://huggingface.co/datasets), but I'm wondering if we should keep consistency here. @merelcht ?

Reply:
Good point, I'd prefer to keep it consistent even this isn't technically a dataset. `HFAutoTransformerDataset` sounds like a good name.